### PR TITLE
Update djtest

### LINKED ISSUE #1: Add ptvsd as dependency
This pip package allows for remote debugging into running container.

### PR DESCRIPTION
**Pull request by A-Baji**

### CODE REVIEW

**Comment by A-Baji** [comment]
Supersedes #4

File: dist/alpine/Dockerfile [hunk 1]
```diff
@@ -1,34 +1,37 @@
 ARG BASE_IMAGE_HASH
 ARG PY_VER
+FROM datajoint/djbase:py${PY_VER}-alpine-${BASE_IMAGE_HASH} as test_build
 
-FROM datajoint/djbase:py${PY_VER}-alpine-${BASE_IMAGE_HASH} as jup_build
-USER dja:anaconda
-COPY --chown=dja:anaconda ./config/pip_requirements.txt /tmp/
+LABEL maintainerName="Raphael Guzman" \
+	maintainerEmail="raphael@datajoint.com" \
+	maintainerCompany="DataJoint"
```
Comment (guzman-raphael) [inline dist/alpine/Dockerfile]: You can update this to be more generic than calling me out specifically. :sweat_smile: 

Perhaps:

```suggestion
LABEL maintainerName="DataJoint Contributors" \
	maintainerEmail="service-health@datajoint.com" \
	maintainerCompany="DataJoint"
```

File: dist/alpine/Dockerfile [hunk 2]
```diff
@@ -1,34 +1,37 @@
...
 ENV APK_REQUIREMENTS /tmp/apk_requirements.txt
 ENV PIP_REQUIREMENTS /tmp/pip_requirements.txt
 ENV CONDA_REQUIREMENTS /tmp/conda_requirements.txt
-ENV PATH "/home/dja/.local/bin:/opt/conda/bin:$PATH"
 ENTRYPOINT ["/entrypoint.sh"]
 WORKDIR /main
 VOLUME /tmp/.X11-unix
-CMD ["/bin/sh", "-l"]
\ No newline at end of file
+EXPOSE 8888
```
Comment (guzman-raphael) [inline dist/alpine/Dockerfile]: We can remove this. We use this port for serving Jupyter. Not applicable here.

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@A-Baji Exciting stuff! :star_struck: We are getting so close to having things ready for updating our test images. :+1: 

Suggesting only minor corrections.

**Review by guzman-raphael — APPROVED** [review]

**Pull request merged**